src/__tests__/config.module.spec.ts

```diff
@@ -49,6 +49,109 @@ describe('Config Nest Module', () => {
     expect(componentTest.testConfig(null)).toEqual({ port: 2000 });
   });
 
+  it('Multiple ConfigParam decorators', async () => {
+    @Injectable()
+    class ComponentTest {
+      constructor() {}
+
+      @Configurable()
+      testConfig(
+        @ConfigParam('config.server') server: any,
+        @ConfigParam('config.stub') stub: any
+      ) {
+        return { serverPort: server.port, stubPort: stub.port };
+      }
+    }
+
+    const module = await Test.createTestingModule({
+      imports: [
+        ConfigModule.load(path.resolve(__dirname, '__stubs__', '**/*.ts')),
+      ],
+      providers: [ComponentTest],
+    }).compile();
+    const componentTest = module.get<ComponentTest>(ComponentTest);
+    expect(componentTest.testConfig(null, null)).toEqual({ serverPort: 2000, stubPort: 2000 });
```

Comment:
As a side note, to get rid of `null`, `null`
You could make the parameters optional `?`
So you don't have to to pass it to the method.